### PR TITLE
MMT-1876: Link SKW, Data Centers, instruments, and platforms in a collection's preview to Earthdata Search collections with the same keywords

### PR DESCRIPTION
**Pull request by ChristianTrummer99**
- added an `edsc_url` method to read the current MMT environment (sit,prod,uat,etc) and return the proper edsc url for passage to the preview gem

### CODE REVIEW

File: app/helpers/collections_helper.rb [hunk 1]
```diff
@@ -41,4 +41,19 @@ def nrt_badge(metadata)
       content_tag(:span, 'NRT', class: 'eui-badge nrt')
     end
   end
+
+  def edsc_url
+    config_services = Rails.configuration.services
+    if Rails.env.production?
+      config_services.dig('earthdata', 'ops', 'edsc_root')
+    elsif Rails.env.sit? || Rails.env.development? || Rails.env.test?
```
Comment (jdf-eos) [inline app/helpers/collections_helper.rb:49]: Is it possible to run a local EDSC?

Comment (ChristianTrummer99) [reply app/helpers/collections_helper.rb:49]: I dont think so (just trying https://search.localtest.earthdata.nasa.gov/search doesn't work) but @ryanmiller-1 @charleshuang80 may have insight.

Comment (ryanmiller-1) [reply app/helpers/collections_helper.rb:49]: EDSC has instructions in their repo to run locally.  I don't think that this work warrants the extra complication to development and testing.

Comment (william-valencia) [reply app/helpers/collections_helper.rb:49]: If you edit your local services.yml, you can just point that location to a local EDSC I believe.

**Pull request merged**